### PR TITLE
Use jtreg 7.3.1+1 for JDK11 tests

### PR DESCRIPTION
**Pull request by llxia**
fixes: https://github.com/eclipse-openj9/openj9/issues/19435

### CODE REVIEW

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/job/Grinder/3561/

File: scripts/getDependencies.xml [hunk 1]
```diff
@@ -30,15 +30,8 @@
...
-				<matches pattern="^11$" string="${JDK_VERSION}"/>
-				<then>
-					<property name="jtregTar" value="jtreg_6_1"/>
-				</then>
-			</elseif>
-			<elseif>
-				<!-- versions 17, 21, 22, 23 -->
-				<matches pattern="^(17|2[123])$" string="${JDK_VERSION}"/>
+				<!-- versions 11, 17, 21, 22, 23 -->
+				<matches pattern="^(11|17|2[123])$" string="${JDK_VERSION}"/>
```
Comment (pshipton) [inline scripts/getDependencies.xml:34]: Why do we need to check versions and fail, requiring an update every time there is a new jdk version? Could it just load 7.3.1.1 until we find a newer jdk version requires something different? The OpenJDK testing checks the version and fails if it's not recent enough.

Comment (llxia) [reply scripts/getDependencies.xml:34]: I see no need to check the version either. I will update the code

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/job/Grinder/3564/

**Review by smlambert — APPROVED** [review]

**Review by karianna — APPROVED** [review]

**Pull request merged**